src/test/fuzz/tx_pool.cpp

```diff
@@ -121,6 +124,19 @@ void MockTime(FuzzedDataProvider& fuzzed_data_provider, const CChainState& chain
     SetMockTime(time);
 }
 
+CTxMemPool MakeMempool(const NodeContext& node)
+{
+    // Take the default options for tests...
+    CTxMemPool::Options mempool_opts{MemPoolOptionsForTest(node)};
+
+    // ...override specific options for this specific fuzz suite
+    mempool_opts.estimator = nullptr;
+    mempool_opts.check_ratio = 1;
```

Comment:
For `.estimator`, it's overriding since `MemPoolOptionsForTest` would have set it.
For `.check_ratio`, yeah it's setting the same value, but it's essentially being explicit in case some day `MemPoolOptionsForTest` is changed.